api/example.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
```

Comment:
I'm wondering if we can group all these client specific artifacts in a single dir. A possibility could be:
```
- api/client
--- example.py
--- patch-client.sh
--- src/  # here we put the generated code
--- patch/
```